Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,238 @@
+
+var randomIndex = Int.random(in: 0..<allTheWords.count)
+let randomWord = allTheWords[randomIndex]
+var hiddenWord: [Character] = Array(repeating: "_", count: randomWord.count)
+let alphabets: Set<Character> = Set("abcdefghijklmnopqrstuvwxyz")
+var guessMax = 6
+var indices: Set<Int> = []
+var lettersThatYouGuessed = [String]()
+var guesses = [Int]()
+
+var guessesIsCorrect = [Int]()
+
+
+
+print(randomWord)
```

Comment:
I am sure that you were using this to test your game but please remember to run and test your code before submitting it. With this line of code, the user sees the randomWord without ever getting a chance to guess it.